src/geo/edge_insets.js

```diff
@@ -102,6 +102,6 @@ class EdgeInsets {
     }
 }
 
-export type PaddingOptions = {top: ?number, bottom: ?number, right: ?number, left: ?number};
+export type PaddingOptions = {+top: ?number, +bottom: ?number, +right: ?number, +left: ?number};
```

Comment:
This hack allows us to assign `Required<PaddingOptions>` to the `PaddingOptions`.